rfcs/0006-cdevents/README.md

```diff
@@ -0,0 +1,135 @@
+# RFC-0006 Flux CDEvents Receiver
+
+<!--
+The title must be short and descriptive.
+-->
+
+**Status:** provisional
+
+<!--
+Status represents the current state of the RFC.
+Must be one of `provisional`, `implementable`, `implemented`, `deferred`, `rejected`, `withdrawn`, or `replaced`.
+-->
+
+**Creation date:** 2023-12-08
+
+**Last update:** 2024-02-09
+
+## Summary
+
+This RFC proposes to add a receiver type to Flux that can receive CDEvents.
+The CDEvents Receiver for Flux receives a CDEvent sent to the receiver webhook URL, verifies it and checks the event type against `.spec.events`, then triggers reconciliation of the configured resources.
```

Comment:
```suggestion
This RFC proposes to add a `receiver` type to Flux that can handle CDEvents.
The `receiver` will receive `CDEvent` events, sent to the receiver's webhook URL. It will verify the received events, check their `event type`, and eventually trigger reconciliation of the configured resources.
```